src/main/resources/project/jython/create_domain.jython

```diff
@@ -1,10 +1,31 @@
 try:
-    connect(r'[% wl_username %]', r'[% wl_password %]', r'[% admin_url %]')
     domainTemplate = r'[% domain_template %]'
     domainDir = r'[% domain_directory %]'
     user = r'[% user %]'
     password = r'[% password %]'
-    createDomain(domainTemplate, domainDir, user, password)
+    listenAddress = r'[% listen_address %]'
+    listenPort = int(r'[% listen_port %]')
+    
+    print('Reading Template - ' + domainTemplate)
+    readTemplate('C:\Oracle\Middleware\Oracle_Home\user_projects\DemoTemplate.jar')
```

Comment:
We should not have the path hard-coded here. It should be read from a parameter value.